core/client-internal/src/test/java/tachyon/client/file/FileInStreamTest.java

```diff
@@ -67,6 +67,7 @@
   private TachyonBlockStore mBlockStore;
   private FileSystemContext mContext;
   private FileInfo mInfo;
```

Comment:
could we just replace mInfo with mStatus here? having both seems redundant and error-prone